src/common/compute/backends/sciserver-compute/Client.js

```diff
@@ -52,7 +52,7 @@ define([
             userVolumeId: volume.id,
             needsWriteAccess: SciServerClient.isWritable(volume),
         }));
-        const filepath = `/home/idies/workspace/Storage/${dirname}`;
+        const filepath = `/home/idies/workspace/Temporary/${dirname}`;
```

Comment:
Will this work if there are no volumes in `Temporary` except `scratch`?

Reply:
It depends what the user specified as the volume to use. If the user specified a volume in the "Temporary" volume pool, then it will work. It worked the same before except the volume had to be in "Storage".

However, this could probably be removed and it could just use scratch by default.